src/mavis_config/config.json

```diff
@@ -489,6 +489,11 @@
             "description": "skip the validation stage of the MAVIS pipeline",
             "type": "boolean"
         },
+        "summary.cluster_radius": {
+            "default": 10,
+            "description": "Maximum distance allowed between breakpoint pairs of called SVs and those of known SVs",
```

Comment:
you may need to be more explicit here, like "Distance used in matching input SVs to reference SVs through clustering"